gifify.sh

```diff
@@ -16,12 +17,24 @@ function printHelpAndExit {
   echo '               below approximately 60.'
   echo '  p SCALE:     Rescale the output, e.g. 320:240'
   echo '  x:           Remove the original file and resulting .gif once the script is complete'
+  echo '  f from:      The start time of the clip, in seconds'
```

Comment:
You'll need to add the `to=` and `from=` vars, otherwise you'll get:
```
/usr/local/bin/gifify: line 104: to: unbound variable
/usr/local/bin/gifify: line 104: temp: unbound variable
```